src/pages/home.js

```diff
@@ -67,11 +77,7 @@ export function Home() {
 
   // effect to initially fetch count when component mounts
   useEffect(() => {
-    if (
-      !triedFetchedBenefitsCount &&
-      !isFetchingBenefitsCount &&
-      !fetchBenefitsCountFailed
-    ) {
+    if (!triedFetchedBenefitsCount && !isFetchingBenefitsCount) {
```

Comment:
Why is this being modified. This is related to other tasks. Does this need to be modified for keycloak ?

Reply:
good catch

Reply:
no,  lifejourney were removed from the main, and after some git problems I redo everything